builds/misc/images-mqtt.yaml

```diff
@@ -3,6 +3,7 @@ trigger:
   branches:
     include:
       - master
+      - release/*
```

Comment:
nit: that won't hurt, but not really necessary. This pipeline (and image) is used for MQTT perf tests. And we are fine testing master only.

Reply:
Do you think it would be a reasonable thing to do to have this pipeline be run as well? If not, I'll just remove it.

Reply:
There is no use for now to run it on release. We don't ship any artifacts produced by this pipeline.